tests/functions.rs

```diff
@@ -644,4 +664,4 @@ compile_pass! {
 
         fallible(): void;
     })
-}
+}*/
```

Comment:
what's the commented code for?

Reply:
probably needs to be fixed in the future